docs/nodes/configuration.md

```diff
@@ -198,35 +221,59 @@ pprof-laddr = ""
 #######################################################
 [p2p]
 
+# Enable the new p2p layer.
+use-legacy = false
+
+# Select the p2p internal queue
+queue-type = "priority"
+
 # Address to listen for incoming connections
 laddr = "tcp://0.0.0.0:26656"
 
 # Address to advertise to peers for them to dial
 # If empty, will use the same port as the laddr,
 # and will introspect on the listener or use UPnP
-# to figure out the address.
+# to figure out the address. ip and port are required
+# example: 159.89.10.97:26656
 external-address = ""
 
 # Comma separated list of seed nodes to connect to
+# We only use these if we can’t connect to peers in the addrbook
+# NOTE: not used by the new PEX reactor. Please use BootstrapPeers instead.
+# TODO: Remove once p2p refactor is complete
+# ref: https:#github.com/tendermint/tendermint/issues/5670
 seeds = ""
 
+# Comma separated list of peers to be added to the peer store
+# on startup. Either BootstrapPeers or PersistentPeers are
+# needed for peer discovery
+bootstrap-peers = ""
```

Comment:
I'm not too sure if we want to put this here, but the other day I was thinking that if someone knows a few nodes that are producing snapshots, they should populate this field with their nodeID's so that they connect with them first. This is kind of like a short term service discovery thing